.circleci/config.yml

```
@@ -138,4 +140,4 @@ workflows:
           filters:
             branches:
               only:
-                - master
+                - master
```

```suggestion
                - master

```
Please keep newline at the end of the file (linux/unix standard)